ament_tools/build_types/cmake.py

```diff
@@ -167,8 +167,9 @@ def _common_cmake_on_build(self, should_run_configure, context, prefix, extra_cm
             solution_file = solution_file_exists_at(
                 context.build_space, context.package_manifest.name)
             cmd = prefix + [MSBUILD_EXECUTABLE]
-            if '-j1' in context.make_flags:
+            if not '-j1' in context.make_flags:
                 cmd += ['/m']
+                os.environ['CL'] = '/MP'
```

Comment:
I changed it so that the BuildAction can take a custom env. This is slightly worse than putting the env in the `.bat` file that prefixes the command, because it's less obvious to the user that this flag has been switched on. Anyways, if we agree it's better, we can merge https://github.com/ament/ament_tools/pull/101 into this pr.